README.md

```diff
@@ -22,7 +22,10 @@ sudo docker build . -t sydneyinformaticshub/rnaseq-rstudio:4.1.0
 
```

Comment:
On my Mac,  I was unable to build the docker container. See my terminal below:

<img width="1306" alt="image" src="https://github.com/Sydney-Informatics-Hub/Rstudio-rnaseq-contained/assets/50643975/08845da5-744f-4037-a0a5-a6721a566060">

Reply:
Issue flagged here. https://github.com/Sydney-Informatics-Hub/Rstudio-rnaseq-contained/issues/9#issue-1948647023